src/private/CABasicAnimation+MotionAnimator.m

```diff
@@ -180,7 +181,9 @@ void MDMConfigureAnimation(CABasicAnimation *animation, MDMMotionTiming timing)
       // As for our sign, if absoluteInitialVelocity matches the direction of displacement, then our
       // sign will be positive. Otherwise, our sign will be negative, as expected by Core Animation.
 
-      springAnimation.initialVelocity = absoluteInitialVelocity / displacement;
+      if (fabs(displacement) > 0.00001) {
```

Comment:
I don't think tgmath.h is required if you're going to (potentially) promote `displacement` to a double and then check the double return value.

Reply:
Done.